blocks/library/shortcode/index.js

```diff
@@ -2,14 +2,143 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { withInstanceId, Dashicon } from '@wordpress/components';
+import { withInstanceId, Dashicon, Spinner, SandBox } from '@wordpress/components';
+import { Component, compose } from '@wordpress/element';
+import { addQueryArgs } from '@wordpress/url';
+import BlockControls from '../../block-controls';
+import { getCurrentPostId } from '../../../editor/store/selectors';
+import { connect } from 'react-redux';
```

Comment:
Might be good to separate the external, internal and WordPress dependencies like we do in other files

Reply:
Agree, I will separate them out. Thanks for bringing to my attention.